mesonbuild/interpreterbase.py

```diff
@@ -403,19 +403,19 @@ def bool_method_call(self, obj, method_name, args):
         (posargs, _) = self.reduce_arguments(args)
         if method_name == 'to_string':
             if len(posargs) == 0:
-                if obj == True:
+                if obj is True:
```

Comment:
Obj is always boolean here so the check (and the two other below) should be `if obj`.